pkg/config/config_template.yaml

```diff
@@ -633,6 +633,12 @@ api_key:
   #
   # min_collection_interval: 15
 
+  ## @param namespace - string - optional - default: default
+  ## Namespace can be used to disambiguate devices with same IPs.
+  ## Changing namespace will cause devices being recreated in NDM app.
+  #
+  # namespace: default
```

Comment:
Probably needs to be declared in `config.go`

Reply:
Good catch, thank you